content/en/docs/tasks/manage-daemon/update-daemon-set.md

```diff
@@ -33,7 +33,7 @@ DaemonSet has two update strategy types:
 * RollingUpdate: This is the default update strategy.  
   With `RollingUpdate` update strategy, after you update a
   DaemonSet template, old DaemonSet pods will be killed, and new DaemonSet pods
-  will be created automatically, in a controlled fashion.
+  will be created automatically, in a controlled fashion. At most one pod will be running on each node during the whole update process.
```

Comment:
I think it's worth pointing out that the “at most one” applies to Pods from the DaemonSet, and not to the set of Pods running via that Node's kubelet.

Reply:
clarified this in new commit